src/IssueViz.UnitTests/Selection/IssueFlowStepNavigatorTests.cs

```diff
@@ -247,50 +247,6 @@ public void GotoPreviousNavigableFlowStep_NavigatesToPreviousNavigableLocation()
             VerifyNavigation(expectedNavigation);
         }
 
-        [TestMethod]
```

Comment:
No longer relevant, there is no property to set anymore, and the span is set in LocationNavigator